src/SDKs/Storage/Management.Storage/Microsoft.Azure.Management.Storage.csproj

```diff
@@ -4,7 +4,7 @@
     <PackageId>Microsoft.Azure.Management.Storage</PackageId>
     <Description>Microsoft Azure Management Storage Library</Description>
     <AssemblyName>Microsoft.Azure.Management.Storage</AssemblyName>
-    <Version>6.3.0-preview</Version>
+    <Version>6.4.0-preview</Version>
     <PackageTags>Microsoft Azure Storage management;Storage;Storage management</PackageTags>
     <NoWarn>$(NoWarn);CS1591</NoWarn>
```

Comment:
@JasonYang-MSFT please do not add this to ignore this particular warning.
We will be moving soon where warnings will be treated as errors and especially we do not want public methods not to have empty xml documentation